packages/server/src/assets/email-templates/reset-password.template.html

```diff
@@ -0,0 +1,300 @@
+                    </tr>
+                  </tbody>
+                </table>
+              </nav>
+              <table
+                class="s-3 w-full"
+                role="presentation"
+                border="0"
+                cellpadding="0"
+                cellspacing="0"
+                style="width: 100%"
+                width="100%"
+              >
+                <tbody>
+                  <tr>
+                    <td
+                      style="
+                        line-height: 12px;
+                        font-size: 12px;
+                        width: 100%;
+                        height: 12px;
+                        margin: 0;
+                      "
+                      align="left"
+                      width="100%"
+                      height="12"
+                    >
+                      &#160;
+                    </td>
+                  </tr>
+                </tbody>
+              </table>
+            </header>
+            <table
+              class="container-fluid"
+              role="presentation"
+              border="0"
+              cellpadding="0"
+              cellspacing="0"
+              style="width: 100%"
+            >
+              <tbody>
+                <tr>
+                  <td
+                    style="
+                      line-height: 24px;
+                      font-size: 16px;
+                      width: 100%;
+                      margin: 0;
+                      padding: 0 16px;
+                    "
+                    align="left"
+                  >
+                    <section>
+                      <table
+                        class="pb-3"
+                        role="presentation"
+                        border="0"
+                        cellpadding="0"
+                        cellspacing="0"
+                      >
+                        <tbody>
+                          <tr>
+                            <td
+                              style="
+                                line-height: 24px;
+                                font-size: 16px;
+                                padding-bottom: 12px;
+                                margin: 0;
+                              "
+                              align="left"
+                            >
+                              <h1
+                                class="text-lg"
+                                style="
+                                  padding-top: 0;
+                                  padding-bottom: 0;
+                                  font-weight: 500;
+                                  vertical-align: baseline;
+                                  font-size: 18px;
+                                  line-height: 21.6px;
+                                  margin: 0;
+                                "
+                                align="left"
+                              >
+                                Aanvraag Password herstellen
```

Comment:
Should be wachtwoord

Reply:
Please change this also in app-router meta title

Reply:
Will do